src/repositories/soft-crud.repository.base.ts

```diff
@@ -168,6 +168,34 @@ export abstract class SoftCrudRepository<
     );
   }
 
+  /**
+   * Method to perform undo the soft delete by Id.
+   * @param id
+   * @param options
+   */
+  async undoSoftDeleteById(id: ID, options?: Options): Promise<void> {
+    const user = await this.findByIdIncludeSoftDelete(id);
+    await this.undoSoftDeleteAll(user);
```

Comment:
here we are passing entity and the target method expects where 
isnt this a type mismatch

Reply:
I have now passed where {id: id}